File: Manuel/2025-08-14/집합/배열/11723.py
import sys
input = sys.stdin.readline

m = int(input())
s = [False] * 21

for _ in range(m):
    line = input().split()
    command = line[0]

    if command == "add":
        x = int(line[1])
        s[x] = True
    elif command == "remove":
        x = int(line[1])
        s[x] = False
    elif command == "check":
        x = int(line[1])
        if s[x]:
            sys.stdout.write("1\n")
        else:
            sys.stdout.write("0\n")
    elif command == "toggle":
        x = int(line[1])
        s[x] = not s[x]
    elif command == "all":
        s = [False] + [True] * 20
    elif command == "empty":
        s = [False] * 21

File: Manuel/2025-08-14/집합/배열/11723.cpp
#include <iostream>
#include <string>
#include <vector>
#define endl "\n"

using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    int m;
    cin >> m;
    vector<bool> s(21, false);

    while(m--)
    {
        string command;
        cin >> command;

        if(command == "add")
        {
            int x;
            cin >> x;
            s[x] = true;
        }
        else if(command == "remove")
        {
            int x;
            cin >> x;
            s[x] = false;
        }
        else if(command == "check")
        {
            int x;
            cin >> x;
            if(s[x])
            {
                cout << 1 << endl;
            }
            else
            {
                cout << 0 << endl;
            }
        }
        else if(command == "toggle")
        {
            int x;
            cin >> x;
            s[x] = !s[x];
        }
        else if(command == "all")
        {
            for(int i = 1; i <= 20; i++)
            {
                s[i] = true;
            }
        }
        else if(command == "empty")
        {
            for(int i = 1; i <= 20; i++)
            {
                s[i] = false;
            }
        }
    }
    return 0;
}